cli/cli/helpers/portal_manager/portal_manager.go

```diff
@@ -207,7 +207,7 @@ func (portalManager *PortalManager) MapPorts(ctx context.Context, localPortToRem
 		} else {
 			logrus.Warnf("Mapping other than TCP or UDP port is not supported right now. Will skip port '%d' because protocal is '%v'", remotePort.GetNumber(), remotePort.GetTransportProtocol())
 		}
-		forwardPortsArgs := portal_constructors.NewForwardPortArgs(uint32(localPort), uint32(remotePort.GetNumber()), &transportProtocol)
+		forwardPortsArgs := portal_constructors.NewForwardPortArgs(uint32(localPort), uint32(remotePort.GetNumber()), kurtosis_context.UserServiceEndpointType, &transportProtocol, &kurtosis_context.ForwardPortDoNotWaitUntilReady)
```

Comment:
We don't need to wait for the port being forwarded for user services.

Reply:
Ah ok, makes sense